src/views/Customer.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h2>お客様の情報を入力してください</h2>
+    <p>性別</p>
+    <input type="radio" id="male" value="男性" v-model="sex">
+    <label for="male">男性</label>
+    <input type="radio" id="female" value="女性" v-model="sex">
+    <label for="female">女性</label>
+    <br>
+    <p>生年月日</p>
+    <select v-model="year">
+      <option v-for="year in years" :key="year">{{year}}</option>

```

Comment:
`{{}}`の両内側はスペースを空けましょう。
他の部分も同様に修正しましょう。
エディターの自動整形を使うといいですね。
```suggestion
      <option v-for="year in years" :key="year">{{ year }}</option>

```